src/sentry/static/sentry/app/components/similarSpectrum.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import styled from '@emotion/styled';
+
+import {t} from 'app/locale';
+
+type Props = {
+  className?: string;
+};
+
+function SimilarSpectrum({className}: Props) {
+  return (
+    <div className={className}>
+      <span>{t('Similar')}</span>
+      <SpectrumItem colorIndex={4} />
+      <SpectrumItem colorIndex={3} />
+      <SpectrumItem colorIndex={2} />
+      <SpectrumItem colorIndex={1} />
+      <SpectrumItem colorIndex={0} />
+      <span>{t('Not Similar')}</span>
+    </div>
+  );
+}
+
+const StyledSimilarSpectrum = styled(SimilarSpectrum)`
+  display: flex;
+  font-size: ${p => p.theme.fontSizeSmall};
+`;
+
+type ItemProps = {
+  colorIndex: number;
```

Comment:
I wonder if you could finagle `keyof typeof theme.similarity.colors`

Reply:
Alternatively, I think @doralchan suggested we not have these colors in the theme file at all?